conf.py

```diff
@@ -54,7 +54,7 @@
 CANNED_MESSAGES = {
     "codeblock": (
         "Increase your chance of getting help and look like a pro by sharing codeblocks not images. "
-        "For example, you can type the following. Note, the \\`\\`\\` must be on their own line.\n"
+        "For example, you can type the following. =Note, the \\`\\`\\` must be on their own line, preferrably with a language name for source highlighting.\n"
```

Comment:
```suggestion
        "For example, you can type the following. Note, the \\`\\`\\` must be on their own line, preferably with a language name for source highlighting.\n"
```

WIthout an example, this might be more confusing than helpful.

I don't think we should _only_ show codeblocks with a language to the exclusion of a codeblock without a language ... and two examples might be a bit much.

Reply:
The linked page shows both examples, though.  But I have a feeling that people do not read past the first examples there, since it seldom happens that they put a language block there. Also, "plain" or "text" is valid, and they may not know that either, but it is also not mentioned in the link.

Reply:
This note is often peoples' first introduction to codeblocks. Even with this note, we often see people struggle to figure out how to use codeblocks. We want this note to be as simple and easy to follow as possible. I'm concerned that adding a note about adding a language makes things more confusing, especially without any example of what it looks like. I believe that makes this change not aligned with the goal here.

Reply:
I'm concerned about adding too much here. What do you think about changing the last sentence instead?

`"Click here to learn more about codeblocks, like adding syntax highlighting: ..."`